core/shared/src/main/scala-2/zio/prelude/Newtype.scala

```diff
@@ -225,7 +225,7 @@ abstract class Newtype[A] extends NewtypeVersionSpecific {
 
   /**
    * This method is used to generate Newtype that can be validated at
-   * compile-time. This must wrap a [[Assertion]] and be assigned to
```

Comment:
I'm not sure what broke this link in the scaladoc.
After this pr it started failing with `Could not find any member to link for "Assertion".`  for 2.11. All other scala versions are fine.

Reply:
Scala's weird sometimes :D